### PR TITLE
Kamila/remove handle

### PR DESCRIPTION
**Pull request by kamilababayeva**
This PR removes user's handle from the code. Handle has the same meaning as username, so it is useless to have both entities. For later, we can enforce to have only username which will be unique.

### CODE REVIEW

File: app/src/main/java/ch/sdp/vibester/model/UserSharedPref.kt [hunk 1]
```diff
@@ -159,7 +138,7 @@ class UserSharedPref private constructor() {
                 correctSongs = sharedPref.getInt(CORRECT_SONGS, 0)
                 ranking = sharedPref.getInt(RANKING, 0)
             }
-            return User(handle, username, image, email, totalGames, bestScore, correctSongs, ranking)
+            return User( username, image, email, totalGames, bestScore, correctSongs, ranking)
```
Comment (jiabaow) [inline app/src/main/java/ch/sdp/vibester/model/UserSharedPref.kt:141]:
```suggestion
            return User(username, image, email, totalGames, bestScore, correctSongs, ranking)

```

File: app/src/androidTest/java/ch/sdp/vibester/activity/ScoreBoardActivityTest.kt [hunk 2]
```diff
@@ -48,16 +48,4 @@ class ScoreBoardActivityTest {
             )
         }
     }
-
-    @Test
```
Comment (jiabaow) [inline app/src/androidTest/java/ch/sdp/vibester/activity/ScoreBoardActivityTest.kt:52]: Why this test is removed? Does it relate to handle?

**Review by jiabaow — COMMENTED** [review]
Thanks for removing the duplicate element!
I left a question that is not very clear to me.

File: app/src/androidTest/java/ch/sdp/vibester/activity/ScoreBoardActivityTest.kt (hunk 2 above)
Comment (kamilababayeva) [reply app/src/androidTest/java/ch/sdp/vibester/activity/ScoreBoardActivityTest.kt:52]: The order of the users in the firebase is not deterministic. We have been deleting/adding users, so the user positions always change. I believe the removal of the test will not decrease any coverage of your changes

**Review by jiabaow — APPROVED** [review]
👍

**Pull request merged**